dandi/pynwb_utils.py

```diff
@@ -382,13 +392,15 @@ def validate(path: str | Path, devel_debug: bool = False) -> list[ValidationResu
             raise
         errors.append(
             ValidationResult(
-                origin=ValidationOrigin(
-                    name="pynwb",
-                    version=pynwb.__version__,
+                origin=Origin(
+                    validator=Validator.pynwb,
+                    validator_version=pynwb.__version__,
+                    standard=Standard.NWB,
```

Comment:
per discussion, we might not even be able to claim it has anything to do with NWB:
```suggestion
```